pkg/systemd/services/services.jsx

```diff
@@ -160,6 +160,7 @@ class ServicesPageBody extends React.Component {
 
             error: null,
             currentStatus: null,
+            pinnedUnits: localStorage.getItem('pinnedUnits') || [],
```

Comment:
This now returns a string || [] so if we want to load the state here it should use `JSON.parse`.